ci_support/run_docker_build.sh

```diff
@@ -40,7 +40,7 @@ echo "$config" > ~/.condarc
 conda clean --lock
 
 conda update --yes --all
-#conda install --yes conda-build==1.18.2
+conda install --yes conda-build==1.18.2
```

Comment:
`hdfeos2` (and `hdfeos5`) needs `conda-build > 1.18.2` to be able to uncompressed `.Z` files.